src/lumigo_tracer/lumigo_utils.py

```diff
@@ -547,21 +549,38 @@ def _recursive_omitting(
     d, free_space = prev_result
     if free_space < 0:
         return d, free_space
+    should_skip_key = False
+    if isinstance(d, list):
+        d.append(None)
+        key = -1
+    elif omit_skip_path:
+        should_skip_key = omit_skip_path[0] == key
+        omit_skip_path = omit_skip_path[1:] if len(omit_skip_path) > 1 and should_skip_key else None
```

Comment:
```suggestion
    elif omit_skip_path:
        should_skip_key = omit_skip_path[0] == key
        omit_skip_path = omit_skip_path[1:] if should_skip_key else None
```
Is more elegant IMO :)
(empty lists are falsy just like None)

Reply:
I was also thinking about it but I think it might be confusing because `[]` might be interpreted as "No more items in `omit_skip_path` so we arrived to the place that should be skipped" or in other words: "when the list is empty, we walked all the path, arrived at the destination and it's time to scrub".

Reply:
yeah, but that's not how the function works 🤔 
If you'll give an empty list to the function - what should it return?